Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -668,4 +674,32 @@ def test_search
   ensure
     desc_cache.unlink
   end
+
+  def test_bundle
+    skip_unless_test_cmd_taps
+    HOMEBREW_REPOSITORY.cd do
+      shutup do
+        system "git", "init"
+        system "git", "commit", "--allow-empty", "-m", "This is a test commit"
+      end
+    end
+
+    mktmpdir do
+      FileUtils.touch "Brewfile"
+      assert_equal "The Brewfile's dependencies are satisfied.",
+        cmd("bundle", "check")
+    end
+  ensure
+    (HOMEBREW_REPOSITORY/".git").rmtree
```

Comment:
This is causing trouble, as it is also executed if you skip the test due to the `skip_unless_test_cmd_taps` at the top of this test method.